### PR TITLE
(feat) added benchmarks for a simple Go installation

### PR DESCRIPTION
**Pull request by kevincobain2000**
Woah, we have a lot of allocs. Thanks to your new mock tests we can now measure them.

### CODE REVIEW

File: .github/workflows/coveritup.yml [hunk 1]
```diff
@@ -41,6 +41,13 @@ jobs:
           type: unit-test-run-time
           command: go test -race -v ./... -count=1 -coverprofile=coverage.out
           record: runtime
+      - name: Benchmark Test
```
Comment (kevincobain2000) [inline .github/workflows/coveritup.yml:44]:
``` 
╰─$ go test  -count 2 -bench=. ./... -benchmem|grep "allocs"
     145	   8069938 ns/op	 7360722 B/op	   19794 allocs/op
     152	   8272545 ns/op	 7370076 B/op	   19794 allocs/op
```

File: gobrew_test.go [hunk 2]
```diff
@@ -25,6 +25,17 @@ func setupGobrew(t *testing.T, ts *httptest.Server) GoBrew {
 	return gb
 }
 
+func BenchmarkInstallGo(t *testing.B) {
+	for i := 0; i < t.N; i++ {
+		ts := httptest.NewServer(http.FileServer(http.Dir("testdata")))
```
Comment (juev) [inline gobrew_test.go:30]: why do you include the launch of the web server in the performance test?
in my opinion, this should not be done in a loop, but at the very beginning of the function, after which we should reset the time counter.
in this case, the data will be correct.

**Review by juev — CHANGES_REQUESTED** [review]
I think we need to correct the test

File: gobrew_test.go (hunk 2 above)
Comment (kevincobain2000) [reply gobrew_test.go:30]: Ah. Thanks!!

**Review by juev — APPROVED** [review]

**Pull request merged**